lib/phut/phost.rb

```diff
@@ -22,7 +25,7 @@ def stop
     private
 
     def pid_file
-      "#{Phut.settings['PID_DIR']}/phost.#{@ip}.pid"
+      "#{Phut.settings['PID_DIR']}/phost.#{name}.pid"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.